### PR TITLE
feat: Break out `host`, `user`, `port`, etc. into individual config values

### LINKED ISSUE #30: Break out host/port/username configurations

### PR DESCRIPTION
**Pull request by visch**
Closes #30, #5

- [x] Need to implement SSL as a configuration option as well (Going to do separately see https://github.com/MeltanoLabs/tap-postgres/issues/117 )

### CODE REVIEW

File: tap_postgres/tap.py [hunk 1]
```diff
@@ -5,31 +5,90 @@
...
+            "user",
+            th.StringType,
+            description=(
+                "User name used to authenticate. "
+                + "Note if sqlalchemy_url is set this will be ignored."
+            ),
+        ),
+        th.Property(
+            "password",
+            th.StringType,
```
Comment (pnadolny13) [inline tap_postgres/tap.py:75]:
```suggestion
            th.StringType,
            secret=True,
```

@visch I think I added a similar comment to another PR. I dont think it really makes any functional difference but it helps with how it advertises its metadata to things like meltano and the hub 👍 .

**Review by pnadolny13 — APPROVED** [review]
@visch looks good to me! I made a small suggestion to add the `secret` attribute to the config.

**Comment by edgarrmondragon** [comment]
@visch Are you planning to implement the SSL option in this PR?

**Comment by visch** [comment]
> Member

No, I think we'll just add that, I updated the description for this PR. I'll get this merged. 

Thank you @pnadolny13  / @edgarrmondragon

**Review by edgarrmondragon — APPROVED** [review]

**Pull request merged**